minio/api.py

```diff
@@ -73,6 +71,11 @@
     _APP_INFO.format(__title__,
                      __version__))
 
+# Constants
+MAX_STREAM_OBJECT_SIZE = 10000 * 5 * 1024 * 1024
```

Comment:
if this constant says 50GiB, which is not equal ie 52428800000 != 53687091200 = 50 * 1024 * 1024 * 1024